src/LanguageServer/Protocol/Workspaces/ILspWorkspaceFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.LanguageServer;
+
+/// <summary>
+/// Exposes utilities used for creating various kinds of workspaces.
```

Comment:
this doesn't match what is actually in the interface :)

Reply:
type deleted.